src/algorithm/SORT.cpp

```diff
@@ -21,14 +21,14 @@ namespace algorithm
 SORT::SORT(const RunParams& params)
   : KernelBase(rajaperf::Algorithm_SORT, params)
 {
-  setDefaultSize(1000000);
+  setDefaultProblemSize(1000000);
   setDefaultReps(20);
 
-  setProblemSize( getRunSize() );
+  setTargetProblemSize( getRunProblemSize() );
```

Comment:
Should getRunSize be translated to getTargerProblemSize? It seems like what is being done here is setting the actual problem size that is run for this kernel.

Reply:
Actually, I think I'm going to remove the the target size stuff. It's not used anywhere. I left it in as a stop gap since I didn't want to break anything. Thoughts?

Reply:
That makes sense the target size is really just in the params so kernels can calculate their run size from the params.